tests/integration/targets/inventory_aws_ec2/playbooks/library/test_get_ssm_inventory.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+# Copyright (c) 2023 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = """
+module: test_get_ssm_inventory
```

Comment:
This feels like something we should cleanup and merge into community.aws as `ssm_inventory_info`...